app/src/main/java/com/github/se/wanderpals/model/viewmodel/AdminViewModel.kt

```diff
@@ -70,6 +70,7 @@ open class AdminViewModel(
 
   // Modify the current user's profil photo
   open fun modifyCurrentUserProfilePhoto(profilePhoto: String) {
+    if (!SessionManager.getIsNetworkAvailable()) return // Check if the network is available
```

Comment:
I personally prefer to always but braces in conditional statements but this is just personal preferences.